src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.html

```diff
@@ -0,0 +1,117 @@
+<div class="col-sm-12 col-lg-6">
+  <form class="form-horizontal"
```

Comment:
`form-horizontal` doesn't exists anymore.
```suggestion
  <form class="form"
```